### PR TITLE
fix: use Node16 for Javascript actions

### PR DESCRIPTION
**Pull request by marcind**

### CODE REVIEW

**Review by dlongeuay — DISMISSED** [review]
approved

**Comment by marcind** [comment]
This is blocked by #266 getting merged in.

File: find-path-owners/action.yml [hunk 1]
```diff
@@ -9,5 +9,5 @@ outputs:
   owners:
     description: A JSON-encoded array of matched owners
 runs:
-  using: 'node12'
+  using: 'node16'
   main: 'dist/index.js'
```
Comment (amartin-isp) [inline find-path-owners/action.yml:13]: nit: new line
```suggestion
  main: 'dist/index.js'
  
```

File: tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
+  "extends": "@tsconfig/node16/tsconfig.json",
   "compilerOptions": {
-    "target": "es2015",
-    "moduleResolution": "node"
+    "strict": false,
   }
 }
```
Comment (amartin-isp) [inline tsconfig.json:6]: nit: new line
```suggestion
}

```

**Review by dlongeuay — APPROVED** [review]
approved

**Review by amartin-isp — APPROVED** [review]
lgtm, just have some nits

**Pull request merged**